projects/observability/src/pages/explorer/explorer-dashboard-builder.ts

```diff
@@ -227,10 +239,22 @@ export class ExplorerDashboardBuilder {
               type: 'span-trace-navigation-handler'
             }
           },
+          {
+            type: 'table-widget-column',
+            title: 'Service',
+            width: '20%',
+            value: {
+              type: 'attribute-specification',
+              attribute: 'serviceName'
+            },
+            'click-handler': {
+              type: 'api-trace-navigation-handler'
+            }
+          },
           {
             type: 'table-widget-column',
             title: 'Name',
-            width: '30%',
```

Comment:
For spans, I think it makes more sense to take the width from status - way over provisioned right now since it doesn't have a special renderer. Names can be fairly long.